src/workloads/sharding/WriteOneWithoutShardKeyUnshardedCollection.yml

```diff
@@ -0,0 +1,105 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+Description: |
+  On an unsharded collection on a single shard cluster, run workloads that updateOne, deleteOne, and findAndModify.
+
+GlobalDefaults:
+  Nop: &Nop {Nop: true}
+
+  Database: &Database test
+  UnshardedCollection: &UnshardedCollection Collection1
+
+  DocumentCount: &DocumentCount 10000  # Number of documents to insert and modify.
+
+Actors:
+- Name: CreateShardedCluster
+  Type: AdminCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: admin
+    Operations:
+    - OperationMetricsName: EnableSharding
+      OperationName: AdminCommand
+      OperationCommand:
+        enableSharding: *Database
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: LoadInitialDataOnUnshardedCollection
+  Type: MonotonicSingleLoader
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+    BatchSize: 1000
+    DocumentCount: *DocumentCount
+    Database: *Database
+    Collection: *UnshardedCollection
+    Document: {x: 1, y: 1, z: 1}
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: CRUDOpsOnUnshardedCollection
+  Type: CrudActor
+  Threads: 1  # We want to use 1 thread to avoid updates throwing WriteConflict errors.
+  Phases:
+  - *Nop
+  - *Nop
+  - Repeat: *DocumentCount
+    Database: *Database
+    Collection: *UnshardedCollection
+    Operations:
+    - OperationMetricsName: UpdateOneWithoutShardKeyUnshardedCollection
```

Comment:
Could probably remove the UnshardedCollection suffix since it's already a part of the unsharded collection genny test and the name seems verbose same for the ones below.